pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -406,6 +406,24 @@ type MonitoringSpec struct {
 	// Interval determines prometheus scrape interval
 	// +optional
 	Interval *metav1.Duration `json:"interval,omitempty"`
+
+	// Ceph exporter configuration
+	// +optional
+	Exporter *CephExporterSpec `json:"exporter,omitempty"`
+}
+
+type CephExporterSpec struct {
+	// Only performance counters greater than or equal to this option are fetched
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=10
+	// +kubebuilder:default=5
+	PerfCountersPrioLimit uint `json:"perfCountersPrioLimit,omitempty"`
+
+	// Time to wait before sending requests again to exporter server (seconds)
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=600
+	// +kubebuilder:default=5
+	StatsDelaySeconds uint `json:"statsDelaySeconds,omitempty"`
```

Comment:
In Ceph, this is documented as `--stats-period`, or sometimes `exporter_stats_period`. I agree that `StatsDelaySeconds` is a clearer name, but I think we should consider keeping Rook and Ceph config naming comparable to help users better match Ceph docs to Rook docs.

I would suggest `StatsPeriodSeconds` might be a good value (including units is always a good practice), and it would be helpful to document that the value applies the `exporter_stats_period` config which does return search hits in the upstream Ceph docs. IMO, a link to Ceph docs likely isn't critical for this setting.